JapaneseFilter/sections/specific.txt

```diff
@@ -2871,7 +2871,7 @@ videogakuen.com,yamakamu.net,minaminumaebi.blogmimi.com,hana3939.com,baseballcha
 anichoice.com,sgamesreview.com,shico.xyz,oreteki-design.com###text-14
 av-baron.com,ikikatadatabase.com,videogakuen.com,yasasiikurasi.com,mhmatome.com,sundaygamer.net###text-15
 mangagift.com,videogakuen.com,newageinglog.com,imashun-navi.com,gekikame.com,jumpmatome2ch.biz,news.sod.co.jp###text-16
-xn--eckl0bk7f7cc4od8az005k0ssb.xyz,nemotohiroyuki.jp,sexyvoice.org###text-17
+nihongoka.com,xn--eckl0bk7f7cc4od8az005k0ssb.xyz,nemotohiroyuki.jp,sexyvoice.org###text-17
```

Comment:
The rule is fine, but can you change to `##.widget-single-content-bottom`? (existing: `hentaiandco.com,saitamarket.com,erodou.tousatu.fun,womanlife.online##.widget-single-content-bottom`)

Reply:
OK